fineract-provider/src/main/java/org/apache/fineract/infrastructure/jobs/service/StuckJobExecutorServiceImpl.java

```diff
@@ -102,7 +102,7 @@ protected void doInTransactionWithoutResult(TransactionStatus status) {
 
     private void waitUntilAllPartitionsFinished(Long stuckJobId, String partitionerStepName) throws InterruptedException {
         while (!areAllPartitionsCompleted(stuckJobId, partitionerStepName)) {
-            log.info("Sleeping for a second to wait for the partitions to complete for job {}", stuckJobId);
+            log.info("Sleeping for a second to wait for the partitions to complete for job {} {}", stuckJobId, partitionerStepName);
```

Comment:
This seems not relevant change. Please move it into separate PR!

Reply:
Done! removed